src/libecalc/core/consumers/venting_emitter.py

```diff
@@ -2,75 +2,67 @@
 
 from typing import Dict, List
 
-import numpy as np
-
-from libecalc import dto
 from libecalc.common.logger import logger
 from libecalc.common.temporal_model import TemporalExpression, TemporalModel
-from libecalc.core.models.fuel import FuelModel
+from libecalc.common.units import Unit
+from libecalc.common.utils.rates import TimeSeriesStreamDayRate
 from libecalc.core.result.emission import EmissionResult
-from libecalc.dto import Emission
-from libecalc.dto.types import FuelType
 from libecalc.dto.variables import VariablesMap
 from libecalc.expression import Expression
+from libecalc.presentation.yaml.yaml_types.emitters.yaml_venting_emitter import (
```

Comment:
This import is the wrong way, yaml should 'know' about core, not the other way around.

We are moving towards core not importing from anything else than libecalc.common, it is not the case currently.

We also want to get rid of expressions in core.

Considering what's left in this class if we do that, I suggest just removing this class and put the logic in EnergyCalculator instead. Some of the expression conversion could be done in the yaml model, i.e. yaml_model.get_emission_rate(variables_map) -> returns a TImeSeriesStreamDayRate. Then we can call to_unit(Units.tons) and create an EmissionResult.

Reply:
This venting emitter class in core is removed. Expressions are evaluated in `YamlVentingEmitter` class. A method is generated to return `TimeSeriesStreamDayRate`.